requirements.txt

```diff
@@ -1,4 +1,6 @@
 lxml
+numpy
+scipy
```

Comment:
I'm not sure it works on Linux - scipy had numpy as a build-time dependency for a long time, so just installing them from requirements used not to work. On OS X it may be a bit simpler as wheels are available, but I'm not sure that simply `pip install -r requirements.txt` works either. Travis runs Linux - could you please check that it works? It should be possible to enable Travis for your fork.

Webstruct doesn't actually use scipy and numpy (see https://github.com/scrapinghub/webstruct/issues/21), so another way to handle it is to remove scikit-learn dependency. It could be more work though.